packages/runtime/container-runtime/src/gc/garbageCollection.md

```diff
@@ -48,12 +48,75 @@ In this phase, the GC algorithm identifies all Fluid objects that are unreferenc
+as part of the Mark Phase when Sweep is disabled. In this mode, any object that GC determines is ready to be deleted is
+marked as a "Tombstone", which triggers certain logging events and/or behavior changes if/when that Tombstoned object is
+accessed by the application.
+
+## GC Configuration
+
+The default configuration for GC today is:
+
+-   GC Mark Phase is **enabled**, including Tombstone Mode
+-   Session Expiry is **enabled**
+-   GC Sweep Phase is **disabled**
+
+### Techniques used for configuration
+
+There are two ways to configure the Fluid Framework's GC behavior, referred to by name throughout these documents:
+
+1.  **"GC Options"**: `ContainerRuntime.loadRuntime` takes an options value of type `IContainerRuntimeOptions`.
+    This type includes a sub-object `gcOptions`, for GC-specific options.
+2.  **"Config Settings"**: The `Loader`'s constructor takes in `ILoaderProps`, which includes `configProvider?: IConfigProviderBase`
+    This configProvider can be used to inject config settings.
+
+Typically GC Options are used for more "official" and stable configuration, whereas Config Settings provide a mechanism
+for apps to override settings easily, e.g. by backing their `IConfigProviderBase` with a configuration/flighting service.
+Additionally, FF will fallback to reading from Local/Session Storage if the provider doesn't mention a particular
+Config Setting, making it convenient to override these while debugging.
+
+### Disabling Mark Phase
+
+If you wish to disable Mark Phase for newly-created documents, set the `gcAllowed` GC Option to `false`.
+Note that this will disable GC permanently (including the sweep phase) for the container during its lifetime.
+
+Mark Phase can also be disabled just for the session, among other behaviors,
+covered in the [Advanced Configuration](./gcEarlyAdoption.md#more-advanced-configurations) docs.
+
+### Enabling Sweep Phase
+
+As explained above, Tombstone Mode can provide a "soft delete" experience to evaluate your app's GC-readiness
+without ultimately jeopardizing user's data.
+
+A full treatment of Tombstone and Sweep configuration can be found in
+[this companion document geared towards early adopters of GC](./gcEarlyAdoption.md).
+
+#### Enabling Tombstone Enforcement
+
+To enable Tombstone Enforcement - meaning, attempting to access a Tombstoned object will fail -
+set the following two Config Settings to `true`:
+
+-   `Fluid.GarbageCollection.ThrowOnTombstoneLoad`
+-   `Fluid.GarbageCollection.ThrowOnTombstoneUsage`
+
+#### Enabling Full Sweep
+
+To enable Sweep Phase for new documents, you must set the `gcSweepGeneration` GC Option to a number, e.g. 0 to start.
+The full semantics of this GC Option are discussed [here](./gcEarlyAdoption.md#differences-between-gcsweepgeneration-and-gctombstonegeneration).
+Note that this will disabled Tombstone Mode.
+
+Additionally, you must set these two Config Settings to `true` in the session:
+
+-   `Fluid.GarbageCollection.Test.SweepDataStores`
```

Comment:
It's weird and inconsistent that we use `Test` or `TestOverride` in some config names. If these are truly meant only for testing, the description around them should mention that. If not, we should rename them.

Reply:
Honestly I don't think it matters, if you care enough to open an ADO item, go for it.  I'm going to focus on the existing small tasks I have already.